src/form.rs

```diff
@@ -39,6 +39,9 @@ use crate::{
     meta_type::MetaType,
 };
 
+use cfg_if::cfg_if;
+#[cfg(all(feature = "std", feature = "schema"))]
```

Comment:
Does `schemars` require std?

Reply:
yep, it depends on `serde` internally and other `std` crates

Reply:
If it depends on `serde` then that will just be brought in automatically?

> and other std crates

What do you mean?

Anyway just wondering what the errors are if you remove the `feature = "std"`. Usually for this type of thing you should be able to configure the features in the manifest.

Reply:
> What do you mean?

It relies on `std` library

Reply:
I initially had it without `feature = "std"` and it errored out that some types can not be found in scope

Reply:
Actually tested without `feature = "std"` and it worked. Idk maybe some odd typo

Reply:
You still need to enable `std` to the `schema` feature (in `Cargo.toml`. I have just tried with `cargo check --no-default-features --features schema` and that fails because of https://github.com/paritytech/scale-info/blob/a69ac5516ab292176a51860ab8c87f89b49ff588/src/lib.rs#L15